### DISCUSSION TITLE
Area aware media_player to use ATTR_MEDIA_ANNOUNCE const

### DISCUSSION THREAD

**Discussion by Jhonattan-Souza** [jseidl/magic-areas#407] [Ideas / Feature Requests]
I currently use the music assistant, and in order for the function to play back the audio that was already playing, I need to pass this parameter as true (ATTR_MEDIA_ANNOUNCE)
Do you think that makes sense? If so, I can send the pull request, it's quite simple

**Comment by jseidl** [comment]
This FR is actually reflecting an issue where I don't copy the original service call into the forward call. If I hardcode `ATTR_MEDIA_ANNOUNCE`, then if you decide to send a music to play on Area-aware music player, it will be sent as an announcement instead. 

Thanks for the report, I'll look into it!

**Reply by Jhonattan-Souza** [reply]
> This FR is actually reflecting an issue where I don't copy the original service call into the forward call. If I hardcode `ATTR_MEDIA_ANNOUNCE`, then if you decide to send a music to play on Area-aware music player, it will be sent as an announcement instead.
> 
> Thanks for the report, I'll look into it!

Can only the "data" section of the service be forwarded ?

**Comment by jseidl** [comment]
Yes I just pushed a commit doing that https://github.com/jseidl/hass-magic_areas/pull/414/commits/1594266f19c41b9ee1603edd002aa8013d5322a7.

Coming up next release!
